README.md

```diff
@@ -111,38 +73,344 @@ requirements that this script has that may not already be in place:
 If you are running Debian or Ubuntu, you can make sure that these are
 installed by running:
 
-    sudo apt-get install default-jre perl python wget rdiff-backup socat iptables
+`sudo apt-get install default-jre perl python wget rdiff-backup socat iptables`
```

Comment:
Yea, I updated back to it's original form, I just had tried to condense the line down to 80 characters (that's why I added the tick marks since it's shorter than 4 spaces) but I realize it's a bit dumb now that I think about it when the rest of the code formatting uses 4 spaces.